misc/docs.gno.land/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
```

Comment:
Why can't you simply move this file to the other repository?

Reply:
I'm fine not having it... But i don't know if it's will be really confortable to get easy preview of the docs when working on it...

@leohhhn what do you think ?

Reply:
Read this: https://github.com/gnolang/gno/pull/1801#discussion_r1581027727